geemap/geemap.py

```diff
@@ -2436,223 +2438,6 @@ def close_btn_clicked(b):
-
-        Args:
-            latlon (list | tuple): The coordinates (lat, lon) of the point.
-            names (str | list, optional): The names of the layers to be included. Defaults to None.
-            visible (bool, optional): Whether to inspect visible layers only. Defaults to True.
-            decimals (int, optional): Number of decimals to round the pixel values. Defaults to 2.
-            return_node (bool, optional): If True, return the ipytree node.
-                Otherwise, return the ipytree tree widget. Defaults to False.
-
-        Returns:
-            ipytree.Node | ipytree.Tree: The ipytree node or tree widget.
-        """
-        from ipytree import Node, Tree
-
-        if names is not None:
-            if isinstance(names, str):
-                names = [names]
-            layers = {}
-            for name in names:
-                if name in self.ee_layer_names:
-                    layers[name] = self.ee_layer_dict[name]
-        else:
-            layers = self.ee_layer_dict
-        xy = ee.Geometry.Point(latlon[::-1])
-        sample_scale = self.getScale()
-
-        root_node = Node("Pixels", icon="archive")
-
-        nodes = []
-
-        for layer in layers:
-            layer_name = layer
-            ee_object = layers[layer]["ee_object"]
-            object_type = ee_object.__class__.__name__
-
-            if visible:
-                if not self.ee_layer_dict[layer_name]["ee_layer"].visible:
-                    continue
-
-            try:
-                if isinstance(ee_object, ee.ImageCollection):
-                    ee_object = ee_object.mosaic()
-
-                if isinstance(ee_object, ee.Image):
-                    item = ee_object.reduceRegion(
-                        ee.Reducer.first(), xy, sample_scale
-                    ).getInfo()
-                    b_name = "band"
-                    if len(item) > 1:
-                        b_name = "bands"
-
-                    label = f"{layer_name}: {object_type} ({len(item)} {b_name})"
-                    layer_node = Node(label, opened=self._expand_pixels)
-
-                    keys = sorted(item.keys())
-                    for key in keys:
-                        value = item[key]
-                        if isinstance(value, float):
-                            value = round(value, decimals)
-                        layer_node.add_node(Node(f"{key}: {value}", icon="file"))
-
-                    nodes.append(layer_node)
-            except:
-                pass
-
-        root_node.nodes = nodes
-
-        root_node.open_icon = "plus-square"
-        root_node.open_icon_style = "success"
-        root_node.close_icon = "minus-square"
-        root_node.close_icon_style = "info"
-
-        if return_node:
-            return root_node
-        else:
-            return Tree(nodes=[root_node])
-
-    def _objects_info(self, latlon, names=None, visible=True, return_node=False):

```

Comment:
The refactored `_object_info()` function only has the `latlon` parameter, missing the `names` and `visible` parameters. The bug occurs when the mouse-clicking location does not interact the feature collection.